samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -226,6 +230,10 @@ static void radio_config(nrf_radio_mode_t mode, enum transmit_pattern pattern)
 		 */
 		nrf_radio_crc_configure(NRF_RADIO, RADIO_CRCCNF_LEN_Three,
 					NRF_RADIO_CRC_ADDR_SKIP, 0);
+
+		/* preamble, address (BALEN + PREFIX), code indicator, TERM and payload */
+		total_payload_size = 2 + packet_conf.balen + 1 + packet_conf.cilen +
```

Comment:
According to the following link, the preamble in case of the long range phys (125k and 500k) is 10-bytes long ("10 repetitions of the byte _0x3C_").
https://infocenter.nordicsemi.com/topic/ps_nrf52833/radio.html?cp=4_1_0_5_17_0#concept_hz4_cjl_4r

Reply:
Good point. Fixed.